### PR TITLE
Move Next door studios to algolia

### PR DESCRIPTION
**Pull request by gimmeliina**

### CODE REVIEW

**Comment by nrg101** [comment]
why is genderx in this pull request?

there is already an Algolia scraper for genderxfilms.com

File: SCRAPERS-LIST.md [hunk 1]
```diff
@@ -862,15 +862,15 @@ naughtyamerica.com|NaughtyAmerica.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
...
-nextdoormale.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorraw.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoorstudios.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoortaboo.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
-nextdoortwink.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Gay
+nextdoorbuddies.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorcasting.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorebony.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoorhookups.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
+nextdoormale.com|Algolia_NextdoorStudios.yml|:heavy_check_mark:|:x:|:x:|:x:|Python|Gay
```
Comment (bnkai) [inline SCRAPERS-LIST.md:869]: These 2 sites havent migrated to the api yet?
We need to keep them in the Gamma scraper

File: scrapers/Algolia_NextdoorStudios.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: "Next Door Studios"
+sceneByURL:
+  - action: script
+    url:
+      - nextdoorbuddies.com/en/video/
+      - nextdoorcasting.com/
+      - nextdoorebony.com/
+      - nextdoorhookups.com/
+      - nextdoormale.com/
```
Comment (bnkai) [inline scrapers/Algolia_NextdoorStudios.yml:9]: remove them from here, they should remain in gamma

File: scrapers/Algolia_NextdoorStudios.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - validName
+movieByURL:
+  - action: script
+    url:
+      - adulttime.com/en/dvd/
+    script:
+      - python
+      - Algolia.py
+      - nextdoorstudios
+      - movie
```
Comment (bnkai) [inline scrapers/Algolia_NextdoorStudios.yml:47]: no movie sections for the nextdoor sites and the `url` is not correct either

File: scrapers/GammaEntertainment.yml [hunk 4]
```diff
@@ -36,15 +37,6 @@ sceneByURL:
       - mommyblowsbest.com/en/scene/
       - motherfuckerxxx.com/
       - myteenoasis.com/
-      - nextdoorbuddies.com/en/video/
-      - nextdoorcasting.com/
-      - nextdoorebony.com/
-      - nextdoorhookups.com/
-      - nextdoormale.com/
```
Comment (bnkai) [inline scrapers/GammaEntertainment.yml:43]: should not be removed

File: scrapers/GammaEntertainment.yml [hunk 5]
```diff
@@ -174,6 +165,7 @@ xPathScrapers:
                 familycreep: Family Creep
                 fistingcentral: Fisting Central
                 gapingangels: Gaping Angels
+                genderx: Gender X
```
[the remark this answers is not in the document]
Comment (nrg101) [reply scrapers/GammaEntertainment.yml:168]: i'm asking as there is already a scene scraper for genderxfilms.com in https://github.com/stashapp/CommunityScrapers/blob/master/scrapers/Algolia_GenderXFilms.yml

**Comment by Maista6969** [comment]
Closing this in favor of #1437

**Comment by Maista6969** [comment]
Closing this in favor of #1437

**Pull request closed without merging**